TypeCobol/Compiler/CodeElements/Entry/DataType.cs

```diff
@@ -201,15 +201,6 @@ private static Nodes.TypeDefinition CreateBuiltIn(DataType type)
                     false,
                     0,
                     3);
-                var tokenLine = TokensLine.CreateVirtualLineForInsertedToken(dataTypeDescriptionEntry.Line, "01 CURRENCY TYPEDEF STRICT PUBLIC PIC X(03).");
```

Comment:
Why remove this code?

Reply:
While tracking usages of `CreateVirtualLineForInsertedToken` to update the param list, I noticed that CURRENCY is the only built-in type with 'fake' tokens. So I removed it for sake of consistency. I think ConsumedTokens for CURRENCY may have been required in the past but are no longer used now (?)

Reply:
yes done